contracts/scripts/interactions/InteractRiverRegistry.s.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+// interfaces
+import {IRiverConfig} from "contracts/src/river/registry/facets/config/IRiverConfig.sol";
+
+// libraries
+
+// contracts
+import {Interaction} from "contracts/scripts/common/Interaction.s.sol";
+
+contract InteractRiverRegistry is Interaction {
+  function __interact(address deployer) internal override {
+    address riverRegistry = getDeployment("riverRegistry");
+
+    uint64[] memory chains = new uint64[](11);
+    chains[0] = 1; // mainnet
+    chains[1] = 11155111; // sepolia
+    chains[2] = 550; // river mainnet
+    chains[3] = 6524490; // river testnet
+    chains[4] = 8453; // base
+    chains[5] = 84532; // base sepolia
+    chains[6] = 137; // polygon
+    chains[7] = 42161; // arbitrum
+    chains[8] = 10; // optimism
+    chains[9] = 100; // gnosis
+    chains[10] = 10200; // gnosis chiado
+
+    vm.startBroadcast(deployer);
+    IRiverConfig(riverRegistry).setConfiguration(
+      bytes32("xchain.blockchains"),
```

Comment:
So in the script I wrote, I followed the existing pattern for other config keys and i keccak-encoded `xchain.blockchains`, see https://github.com/river-build/river/blob/90fc274e6d832e2cc71eb0ed84e08f9d3de7a85e/scripts/set-riverchain-config.sh#L32

Can you keccak encode this string and use the resulting bytes32 as the hashed key?

Reply:
why are you encoding it?

Reply:
it takes less bytes to just typecase than to encode it

![Screenshot 2024-09-19 at 12 11 57 PM](https://github.com/user-attachments/assets/975a5b67-cc41-4f7c-af8c-f9530067baba)

Reply:
nvm, realized `xchain.blockchains` is the actual key name and not a namespace path

Reply:
updated in latest push